sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -451,6 +488,154 @@ export class FormTrainingClient {
     await poller.poll();
     return poller;
   }
+
+  /**
+   * Generates authorization for copying a custom model into this Azure Form Recognizer resource.
+   *
+   * @param {string} resourceId Id of the Azure Form Recognizer resource where a custom model will be copied to
+   * @param {string} resourceRegion Location of the Azure Form Recognizer resource
+   * @param {GetCopyAuthorizationOptions} [options={}] Options to get copy authorization operation
+   * @returns {Promise<CopyAuthorization>} The authorization to copy a custom model
+   */
+  public async getCopyAuthorization(
+    resourceId: string,
+    resourceRegion: string,
+    options: GetCopyAuthorizationOptions = {}
+  ): Promise<CopyAuthorization> {
+    const { span, updatedOptions: finalOptions } = createSpan(
+      "FormTrainingClient-getCopyAuthorization",
+      options
+    );
+
+    try {
+      const response = await this.client.generateModelCopyAuthorization(
+        operationOptionsToRequestOptionsBase(finalOptions)
+      );
+      return {
+        resourceId: resourceId,
+        resourceRegion: resourceRegion,
+        //expiresOn: new Date(response.expirationDateTimeTicks),
+        ...(response as CopyAuthorizationResultModel)
```

Comment:
how is this information returned? in a dict like in Python? or is it a CopyAuthorization object?
(sorry, me not understanding too much of JS)

Reply:
It's an object, and in JS properties of an object are like dictionary entries. They can be accessed either by `obj.prop` or `obj["prop"]`,  The result of this can be passed directly to the copy function, or be converted to string, then to object for other libraries to consume.

Reply:
oh cool. Thanks for the clarification!